articles/cognitive-services/Face/QuickStarts/Python.md

```diff
@@ -4,317 +4,141 @@ description: Get information and code samples to help you quickly get started us
 services: cognitive-services
 author: v-royhar
 manager: yutkuo
-
 ms.service: cognitive-services
 ms.technology: face
 ms.topic: article
 ms.date: 06/21/2017
 ms.author: anroth
 ---
 
-# Face API Python Quick Starts
-This article provides information and code samples to help you quickly get started using the Face API with Python to accomplish the following tasks: 
-* [Detect Faces in Images](#Detect) 
-* [Create a Person Group](#Create)
+# Face detection using Cognitive Services
+In this walkthrough, we will the cognitive services [Face APIs](https://azure.microsoft.com/en-us/services/cognitive-services/face/) to detect faces in an image and identify various attributes such as the number of people, their gender and age. The sample images we will use are from the [How-Old Robot](http://www.how-old.net) that uses the same service behind the scenes.
+
+You can run this example as a Jupyter notebook on [MyBinder](https://mybinder.org) by clicking on the launch Binder badge: 
+
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/Microsoft/cognitive-services-notebooks/master?filepath=FaceAPI.ipynb)
 
-Learn more about obtaining free Subscription Keys [here](../../Computer-vision/Vision-API-How-to-Topics/HowToSubscribe.md)
+Please refer to the [API definitions](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395236) for technical documentation for the APIs.
 
-## Detect faces in images with Face API using Python <a name="Detect"> </a>
-Use the [Face - Detect method](https://westcentralus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395236) 
-to detect faces in an image and return face attributes including:
-* Face ID: Unique ID used in several Face API scenarios. 
-* Face Rectangle: The left, top, width, and height indicating the location of the face in the image.
-* Landmarks: An array of 27-point face landmarks pointing to the important positions of face components.
-* Facial attributes including age, gender, smile intensity, head pose, and facial hair. 
+## Prerequisites
 
-#### Face Detect python example request
+You must have a [Cognitive Services API account](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account) with **Face APIs**. The [free trial](https://azure.microsoft.com/en-us/try/cognitive-services/?api=face-api) is sufficient for this quickstart. You need the access key provided when you activate your free trial, or you may use a paid subscription key from your Azure dashboard.
```

Comment:
s/access key/subscription key/g 

Just to unify the used terminology.

Reply:
Done, thanks!